.github/actions/setup_apt/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Setup apt'
+description: 'Setups apt manager with common options'
```

Comment:
do we need it in context of dockerfiles being implemented in parallel?

Reply:
yes, because it would be used in docker build action too and when we use GHA runners